### PR TITLE
Made JAR executable

### PR DESCRIPTION
**Pull request by bruch-alex**
- reduce dependencies in pom file
- make jar executable
- change file reading logic (old one didn't worked with executable JAR)
- Small changes in translations (translation works perfectly now)

### CODE REVIEW

File: src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java [hunk 1]
```diff
@@ -60,46 +61,78 @@ private void setupSalariesListeners() {
...
+     */
+    @FXML
+    private void setupLanguageChangeListener() {
+        yAxis.labelProperty().addListener((_ -> refreshUI()));
+    }
+
+    /**
+     * <p>Although every nameProperty is bound with corresponding StringProperty of Product or Salary visual bug is appearing
+     * so our team decided to refresh entire UI for now. Maybe can fix later.
+     *
```
Comment (nat-laz) [inline src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java:75]: msg to the next  generation? xD

File: src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java [hunk 2]
```diff
@@ -122,10 +155,10 @@ private Product findProductByName(String name) {
...
         return new Tooltip(
-                "Country: " + countryName + "\n" +
-                        "Average Salary: $" + salary + "\n" +
-                        "Product Price: $" + price + "\n" +
-                        "Product Purchasable: " + productsCount
+                L10N.get("country") + ": " + countryName + "\n" +
+                        L10N.get("average_salary") + ": " + salary + "\n" +
+                        L10N.get("product_price") + ": " + price + "\n" +
+                        L10N.get("product_purchasable") + ": " + productsCount
         );
```
Comment (nat-laz) [inline src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java:162]: maybe we should add here and the `productName`? to be more clear

Comment (bruch-alex) [reply src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java:162]: Maybe...

Comment (bruch-alex) [reply src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java:162]: image
image
image

Comment (bruch-alex) [reply src/main/java/org/example/globalgoodsindex/controllers/BarChartController.java:162]: We can also do it like this:
image

**Pull request merged**